mk/common.mk

```diff
@@ -0,0 +1,2 @@
+
+MAKEFLAGS := $(MAKEFLAGS) --no-builtin-rules
```

Comment:
We use `MAKEFLAGS += ` in the sdks, probably better than `:=`? https://github.com/mono/mono/blob/81aec9f6932245fa4f41e0c97e4d11a74d71608e/sdks/wasm/Makefile#L6

Reply:
This doesn't work because it requires `MAKEFLAGS` to be defined already, and for some reasons it isn't.